core/lib/spree/testing_support/dummy_app/assets/config/manifest.js

```diff
@@ -0,0 +1,6 @@
+//
+//= link spree/backend/all.css
+//= link spree/frontend/all.css
+//
+//= link spree/backend/all.js
+//= link spree/frontend/all.js
```

Comment:
I dont think we need to put anything into this file. People could get confused by this and think they need to add this to their app after upgrading to Sprockets 4. By they dont since we put all our assets in the `Rails.application.config.asset.precompile` list and Sprockets 4 still respects this.

Reply:
Yes, I think you are right. It was a sort of experiment and it should work either way. I think that using a manifest coherent with what is required to really use Solidus works better. Thanks!